addon/components/universal-export/template.hbs

```diff
@@ -1,52 +1,32 @@
 {{#modal-view hidden=hidden closeAction=closeModal}}
   <div class="modal-header modal-header__borderless">
     <button type="button" class="close" data-dismiss="modal" aria-label="Close"
-            id='close-x'><span aria-hidden="true">&times;</span></button>
+            id="close-x"><span aria-hidden="true">&times;</span></button>
     <h4 class="modal-title">Export Influencers</h4>
   </div>
 
   <ul class="nav nav-tabs">
-    {{#if tabs.list}}
-        <li class={{if listSelected "active" ""}} role="button" {{action 'setCurrent' 'list'}}>
-          <a>List</a>
-        </li>
-    {{/if}}
-
-    {{#if tabs.campaign}}
-      <li class={{if publishrSelected "active" ""}} role="button" {{action 'setCurrent' 'publishr'}}>
-        <a>Workflow</a>
-      </li>
-    {{/if}}
-
-    {{#if tabs.mailing}}
-      <li class={{if mailingSelected "active" ""}} role="button" {{action 'setCurrent' 'mailing'}}>
-        <a>Emailing</a>
-      </li>
-    {{/if}}
-
-    {{#if tabs.stream}}
-      <li class={{if streamSelected "active" ""}} role="button" {{action 'setCurrent' 'stream'}}>
-        <a>Monitor</a>
+    {{#if tabs.external}}
+      <li {{action "setCurrent" "external"}} class={{if externalSelected "active" ""}} role="button">
+        <a>External</a>
```

Comment:
can we name this tab `In-App`  rather than external.

Reply:
Thaaaanks for that one ! I could not find any wording ! hahah